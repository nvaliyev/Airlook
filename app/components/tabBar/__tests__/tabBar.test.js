import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Tabs from '..';
describe(' Tabbar', () => {
    test('Tabbar snap', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< Tabs/>);
        expect(snapshot).toMatchSnapshot();
    });
});


