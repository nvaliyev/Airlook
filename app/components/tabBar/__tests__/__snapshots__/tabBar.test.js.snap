// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Tabbar Tabbar snap 1`] = `
<LinearGradient
  colors={
    Array [
      "#000000d9",
      "#5d6fc4",
      "#6f347c",
      "#000000d9",
    ]
  }
  end={
    Object {
      "x": 1,
      "y": 0,
    }
  }
  start={
    Object {
      "x": 0,
      "y": 3,
    }
  }
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "height": 40,
    }
  }
>
  <FooterButtons
    ico={
      <Icon
        allowFontScaling={false}
        color="#F7F9FC"
        name="aircraft-take-off"
        size={25}
      />
    }
    onPress={[Function]}
    text="Departure"
  />
  <View
    style={
      Object {
        "backgroundColor": "#F7F9FC",
        "height": 40,
        "opacity": 0.1,
        "width": 1,
      }
    }
  />
  <FooterButtons
    ico={
      <Icon
        allowFontScaling={false}
        color="#F7F9FC"
        name="aircraft-landing"
        size={25}
      />
    }
    onPress={[Function]}
    text="Arrival"
  />
</LinearGradient>
`;
